cmd/export_ledger_entry_changes.go

```diff
@@ -274,23 +281,76 @@ be exported.`,
 func exportTransformedData(
 	start, end uint32,
 	folderPath string,
+	parquetFolderPath string,
 	transformedOutput map[string][]interface{},
 	cloudCredentials, cloudStorageBucket, cloudProvider string,
-	extra map[string]string) error {
+	extra map[string]string,
+	WriteParquet bool) error {
 
 	for resource, output := range transformedOutput {
 		// Filenames are typically exclusive of end point. This processor
 		// is different and we have to increment by 1 since the end batch number
 		// is included in this filename.
 		path := filepath.Join(folderPath, exportFilename(start, end+1, resource))
+		parquetPath := filepath.Join(parquetFolderPath, exportParquetFilename(start, end+1, resource))
 		outFile := mustOutFile(path)
+		var transformedResource []transform.SchemaParquet
+		var parquetSchema interface{}
+		var skip bool
 		for _, o := range output {
 			_, err := exportEntry(o, outFile, extra)
 			if err != nil {
 				return err
 			}
+			switch v := o.(type) {
+			case transform.AccountOutput:
+				transformedResource = append(transformedResource, v)
+				parquetSchema = new(transform.AccountOutputParquet)
+				skip = false
+			case transform.AccountSignerOutput:
+				transformedResource = append(transformedResource, v)
+				parquetSchema = new(transform.AccountSignerOutputParquet)
+				skip = false
+			case transform.ClaimableBalanceOutput:
+				// Skip
```

Comment:
Nvm it was out of original scope for Dune supported tables. I'm fine keeping it out of scope for MVP, but think we should add holistic support at some point

Reply:
Yup I agree. 

I left it out because of the way `ClaimableBalanceOutput` struct is defined. It creates multiple nested structs (like for `claimants`) which didn't nicely stringify